content/post/2020-01-08-cran-error/index.Rmd

```diff
@@ -66,18 +58,25 @@ Installing `Docker` and running a test went well, but what one needs is a VM tha
 ## R and Docker
 
 <!-- r-hub and rocker - which is good for what -->
-There are two services that are worth mentioning:
+There are three services that are worth mentioning:
 
+- [winbuilder](https://win-builder.r-project.org/), which is running on the same Windows machine with the same setup as the CRAN incoming submission checks, works from any operating system, and is very useful for reproducing CRAN errors.  It's also _recommended_ to be used prior to CRAN submissions,
 - [rhub](https://r-hub.github.io/rhub/), which is an R package for using the [R-hub package builder](https://docs.r-hub.io/#package-builder),
 - and the [Rocker project](https://www.rocker-project.org/), which distributes various Docker images useful for R and RStudio testing.
 
 If you are a maintainer of an R package and don't want to install Docker locally, you will find the `rhub` package very useful, as you can push your package to the server for testing with one command! Check out e.g., [the package docs](https://r-hub.github.io/rhub/).
 
+If you want to easily run a check on the Windows system, use the Win-builder webpage or, from R:
+
+```
+devtools::check_win_devel(email = "my_email@address.org")
+```
+
```

Comment:
Why would you use Win-builder instead of R-hub?

Reply:
Yes, I am wondering whether that paragraph is needed since winbuilder is mentioned earlier?

Reply:
(especially since the error here was not on Windows :-) )

Reply:
I didn't say "instead". It was Henrik, who pointed to this tool as being the official Windows checking system for CRAN packages.

Reply:
Oh, ok, that's what you meant - I can remove this one, I agree.

Reply:
Official does not mean that it is the best default for everyone. Maybe it is for him, certainly not for me. Anyway, this is a suggestion, so if you want to leave it in, do so, it is your post. :)

Reply:
I've added the devtools mention near the first mention of the tool and this way you can remove this paragaph :-)